inc/adminfunctions_templates.php

```diff
@@ -88,6 +71,8 @@ function find_replace_templatesets($title, $find, $replace, $autocreate=1, $sid=
 		if($master_template['new_template'] != $master_template['template'])
 		{
 			// Update the rest of our template sets that are currently inheriting this template from our master set
+			$sqlwhere = str_replace(' OR sid=-1', '', $sqlwhere);
```

Comment:
I think this two lines could be better, for instance `array_unique` is unnecessary even if there are repeated SIDs it wouldn't matter.